benchmarks/lobsters/benchmark.rb

```diff
@@ -0,0 +1,44 @@
+require 'harness'
+
+ENV['RAILS_ENV'] ||= 'production'
+ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1' # Benchmarks don't really have 'production', so trash it at will.
+
+Dir.chdir __dir__
+use_gemfile extra_setup_cmd: "bin/rails db:drop db:create && sqlite3 db/production.sqlite3 < db/faked_bench_data.sql"
+
+require_relative 'config/environment'
+require_relative "route_generator"
+
+app = Rails.application
+generator = RouteGenerator.new(app)
+generator.do_login # We want to be logged in for all generated requests
+generator.routes # Make sure routes have been pregenerated
+
+# Track ActiveRecord time
+if ENV['TRACK_AR_TIME']
+  ar_total_duration = 0.0
+  process_start_t = Time.now
+
+  # Track sql.active_record events
+  ActiveSupport::Notifications.subscribe "sql.active_record" do |name, started, finished, unique_id, data|
+    duration = finished - started
+    ar_total_duration += duration
+  end
+
+  at_exit do
+    process_duration = Time.now - process_start_t
+    ar_percent = ar_total_duration * 100.0 / process_duration
+    puts "ActiveRecord time: #{ar_total_duration.round(2)}s (#{ar_percent.round(2)}%) PID: #{Process.pid}"
+  end
+end
+
+run_benchmark(10) do
+  generator.routes.each_with_index do |env, idx|
```

Comment:
Can you add a comment with the path of the source file where the routes are generated?

Reply:
Will do.